server/app/controllers/dev/seeding/DevDatabaseSeedController.java

```diff
@@ -134,178 +101,6 @@ public Result clear() {
   // Create a date question definition with the given name and questionText. We currently create
   // multiple date questions in a single program for testing.
 
```

Comment:
`insertComprehensiveSampleProgram` and `insertMinimalSampleProgram` are moved to `DevDatabaseSeedTask`

Reply:
They also don't need the 

```
 QuestionDefinition sampleNameQuestion =
        databaseSeedTask.run().stream()
            .filter(q -> q.getName().equals("Name"))
            .findFirst()
            .orElseThrow();
```

bit anymore as `DevDatabaseSeedTask` can access `NAME_QUESTION_DEFINITION` from `SampleQuestionDefinitions` directly.